### PR TITLE
Prepare 2.24.0 Release Candidate 0 and update changelog.json

### PR DESCRIPTION
**Pull request by embik**
**What this PR does / why we need it**:
This PR prepares the first 2.24.0 release candidate where we need to bump version. In addition, I've started drafting the changelog.json content (based on https://github.com/kubermatic/kubermatic/pull/12758/), which should be more or less complete now but might see a few changes before the final release.

**Which issue(s) this PR fixes**:

Fixes #

**What type of PR is this?**
/kind documentation
/kind chore

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
NONE
```

**Documentation**:

```documentation
NONE
```

### CODE REVIEW

File: modules/web/src/assets/config/changelog.json [hunk 1]
```diff
@@ -1,161 +1,173 @@
...
     },
     {
       "category": "added",
-      "description": "Add option to disable deployment of default network policies in KubeVirt cluster."
+      "description": "EE only: Add KubeLB integration."
     },
     {
       "category": "added",
-      "description": "Add support for Canal 3.25."
+      "description": "Support for Kubernetes 1.28."
```
Comment (Waseem826) [inline modules/web/src/assets/config/changelog.json:18]: It seems like we already got this covered in the note below. Should we remove this?

Comment (embik) [reply modules/web/src/assets/config/changelog.json:18]: Fair point. Let me remove it!

Comment (embik) [reply modules/web/src/assets/config/changelog.json]: Done.

**Review by Waseem826 — APPROVED** [review]
/approve

**Pull request merged**